pkg/audit/service.go

```diff
@@ -92,15 +92,15 @@ func (service *Service) process(ctx context.Context) error {
 		}
 	}
 
-	verifiedNodes, err := service.Verifier.Verify(ctx, stripe)
-	if err != nil {
-		return err
+	verifiedNodes, verifierErr := service.Verifier.Verify(ctx, stripe)
+	if verifierErr != nil && verifiedNodes == nil {
+		return verifierErr
```

Comment:
why are we being more specific with the errors here?

Reply:
On a similar note, do we need to worry about the error if `verifiedNodes != nil`?

Reply:
Here the idea is that Verify can return some verified notes even if an error occurs. We want to record the audit outcome for these verified nodes. This is why on this line we return `verifyErr` only if there are no successfully verified nodes. Otherwise, we record them and combine `verifierErr` with `reporterErr`... shoot, I have bug on line 103...

Reply:
Line 103 is fixed now.